src/components/InputBook.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/button-has-type */
```

Comment:
- [ ] Kindly note that linters are very important, they help you structure your code very well and prevent future bugs in your code, thus you need to solve them instead of disabling them.